lib/Cake/Test/Case/Controller/Component/PaginatorComponentTest.php

```diff
@@ -629,6 +638,18 @@ public function testPaginateOrderModelDefault() {
 			'PaginatorControllerPost.created' => 'asc'
 		);
 		$this->assertEquals($expected, $result['order']);
+
+		/* CLASSIC */
```

Comment:
If we add comments, they should be a bit more expressive/verbose. Otherweise they aren't much of help.